apps/prairielearn/src/lib/manualGrading.js

```diff
@@ -1,73 +1,134 @@
 // @ts-check
```

Comment:
Presumably we can do a `.js -> .ts` rename in a followup PR with minimal changes?

Reply:
Yup, that's the plan.